src/js/views/maps/FeatureInfoView.js

```diff
@@ -309,24 +310,15 @@ define(
         /**
          * Update the height of the iFrame to match the height of the content
          * within it.
-         * @param {number} [height] The height to set the iFrame to. If no
-         * height is provided, then the height of the content within the iFrame
-         * will be used.
-         * @param {boolean} [limit=true] Whether or not to limit the height of
-         * the iFrame to the height of the window, minus 275px.
          * @since 2.27.0
          */
-        updateIFrameHeight: function (height, limit = true) {
+        updateIFrameHeight: function () {
           const iFrame = this.elements?.iFrame;
-          if (!iFrame) return;
-          if ((!height && height !== 0) || height < 0) {
-            height = iFrame.contentWindow.document.body.scrollHeight + 5;
-          }
-          if (limit) {
-            const maxHeight = window.innerHeight - 275;
-            height = height > maxHeight ? maxHeight : height;
-          }
-          iFrame.style.height = height + "px";
+          const maxHeight = window.innerHeight - 320;
```

Comment:
For "[magic numbers](https://en.wikipedia.org/wiki/Magic_number_%28programming%29)" like 320, can we have a comment associated describing what/why it is this value